config.yaml

```diff
@@ -6,6 +7,7 @@ receivers:
 
   journald:
     units:
+      - microshift
```

Comment:
this will only collector microshift logs which makes the receiver noop in other environments.